cases/forms/reporting.py

```diff
@@ -182,7 +183,12 @@ def clean_search(self):
                 choices.append((addr["value"], addr["label"]))
             self.to_store = {"postcode_results": choices}
         else:
-            results = cobrand.api.geocode(search)
+            try:
+                results = cobrand.api.geocode(search)
+            except requests.HTTPError:
```

Comment:
I wonder if `requests.exceptions.RequestException` would be better to catch, based on [the docs](https://requests.readthedocs.io/en/latest/user/quickstart/#errors-and-exceptions) which mention:

>    All exceptions that Requests explicitly raises inherit from `requests.exceptions.RequestException`

If, for example, the API returned a 200 response taht wasn't valid JSON then L294 of `api.py` would fail and wouldn't be caught here.